libs/sdk-model/src/execution/measure/index.ts

```diff
@@ -203,6 +203,21 @@ export function isSimpleMeasure(obj: any): obj is IMeasure<IMeasureDefinition> {
     return isMeasure(obj) && isMeasureDefinition(obj.measure.definition);
 }
 
+/**
+ * Type guard for checking whether object is an adhoc measure.
+ *
+ * @public
+ */
+export function isAdhocMeasure(obj: any): obj is IMeasure<IMeasureDefinition> {
+    return (
+        isSimpleMeasure(obj) &&
+        (!!obj.measure.definition.measureDefinition.aggregation ||
+            !!obj.measure.definition.measureDefinition.computeRatio ||
+            (Array.isArray(obj.measure.definition.measureDefinition.filters) &&
+                obj.measure.definition.measureDefinition.filters.length > 0))
+    );
```

Comment:
please simplify / decompose this conditional. i suggest something like this:

`
if (!isSimpleMeasure(obj)) return false;

const {measureDefinition} = obj.measure.definition;

return measureDefinition.aggregation || measureDefinition.computeRatio || ..

`

Reply:
Thanks, I've updated the PR.